ui/tests/unit/serializers/mfa-login-enforcement-test.js

```diff
@@ -0,0 +1,33 @@
+import { module, test } from 'qunit';
```

Comment:
thanks for adding a serializer test. we don't do these very often, but it's a great call in this scenario.

Reply:
Thanks! I wanted to be sure that the keys were transformed as expected since I'm not at a place to actually test with a model until the method model work is merged.